homeassistant/components/squeezebox/__init__.py

```diff
@@ -118,7 +119,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: SqueezeboxConfigEntry) -
     device_registry = dr.async_get(hass)
     device = device_registry.async_get_or_create(
         config_entry_id=entry.entry_id,
-        identifiers={(DOMAIN, lms.uuid)},
+        identifiers={(SERVER_DEVICE_ID, lms.uuid)},
```

Comment:
The left side of an identifier should be the domain

Reply:
@joostlek that will result in a merged (incorrect) server and player device when they have a connecting mac, any thoughts on how we can avoid this are welcomed

Reply:
or are you just referring to the naming ie it should be DOMAIN_SERVER say?